config/service.go

```diff
@@ -7,8 +7,10 @@ import (
 
 // CaptionsServiceConfig is the configuration needed to create a CaptionsService
 type CaptionsServiceConfig struct {
-	Server     *server.Config
-	ProjectID  string `envconfig:"PROJECT_ID"`
-	Logger     *log.Logger
-	BucketName string `envconfig:"BUCKET_NAME"`
+	Server         *server.Config
+	ProjectID      string `envconfig:"PROJECT_ID"`
+	Logger         *log.Logger
+	BucketName     string `envconfig:"BUCKET_NAME"`
+	CallbackURL    string `envconfig:"CALLBACK_URL"`
+	CallbackAPIKey string `envconfig:"CALLBACK_API_KEY"`
```

Comment:
See my suggestion about potentially getting rid of these new config parameters, but if that's not possible we can probably have a single parameter called ``CALLBACK_URL`` and make sure that it includes the api key (since we're adding it as a query string parameter anyway).

Reply:
I'll add the api key to the url.